packages/camera/camera_windows/windows/camera_plugin.cpp

```diff
@@ -569,6 +604,66 @@ void CameraPlugin::StopVideoRecordingMethodHandler(
     cc->StopRecord();
   }
 }
+void CameraPlugin::StartImageStreamingMethodHandler(
+            const EncodableMap& args, std::unique_ptr<flutter::MethodResult<>> result) {
+    auto camera_id = GetInt64ValueOrNull(args, kCameraIdKey); //get camera id
```

Comment:
nit: here and elsewhere, comments should follow the Flutter style guide.

Ref: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#documentation-dartdocs-javadocs-etc

Reply:
Here and in the stop method: Given that `camera_id` is a pointer, please make the type explicit to help readers rather than using `auto`.

Reply:
will fix the comments, happy to explicitly type the camera_id however I had left it as auto for consistency with the other pre existing functions.

Reply:
Thanks -- please do. The original camera plugin was written in a relatively short period and not particularly in a style consistent with the rest of Flutter's code/packages.

Generally, when it comes to `auto` we should be following the rules in the Google C++ style guide unless explicitly overridden by our own style guide. When it comes to type, the advice is:

"Use type deduction only if it makes the code clearer to readers who aren't familiar with the project, or if it makes the code safer. Do not use it merely to avoid the inconvenience of writing an explicit type."

The existing code definitely overuses `auto` -- apologies for the confusion.

Ref: https://google.github.io/styleguide/cppguide.html#Type_deduction

Reply:
No worries, I have made the changes to the functions(happy to replicate this to the other similar functions if you are happy with it?). also should the arguments on 289 & 295 be explict?

Reply:
Thanks for checking! For `std::get_if<T>` the type being returned is clear from the template type so adding it to the left hand side doesn't add any more clarity, so I think those two lines are a good example of where to use `auto` and avoid repeating the type.